src/app/shared/services/error-handler.service.ts

```diff
@@ -20,6 +20,7 @@ export class ErrorHandlerService implements ErrorHandler {
         });
         setTimeout(() => { self.messageService.clear(); }, 5000);
 
+        console.error(error);
```

Comment:
I've seen too many cases of this resulting in the output "[object Object]".  So I'd recommend changing this to:
console.error(JSON.stringify(error))

Reply:
Done!